hist/hist/inc/TGraphErrors.h

```diff
@@ -55,6 +55,8 @@ class TGraphErrors : public TGraph {
    TGraphErrors(const TH1 *h);
    TGraphErrors(const char *filename, const char *format="%lg %lg %lg %lg", Option_t *option="");
    ~TGraphErrors() override;
+   virtual void AddPointError(Double_t x, Double_t y, Double_t ex = 0.,
+                              Double_t ey = 0.); ///< Append a new point with errorbars (default to zero) to the graph.
```

Comment:
```suggestion
                              Double_t ey = 0.);
```

Move this one to TGraphErrors.cxx

Reply:
And consider using another rebase with "fixup" of the last commit, so that it matches with the commit message of the current-latest commit, that now does not make sense.
Thanks!